articles/container-registry/container-registry-faq.md

```diff
@@ -250,7 +250,9 @@ Image quarantine is currently a preview feature of ACR. You can enable the quara
 
 ### How do I enable anonymous pull access?
 
-Setting up an Azure container registry for anonymous (public) pull access is currently a preview feature. To enable public access, please open a support ticket at https://aka.ms/acr/support/create-ticket. For details, see the [Azure Feedback Forum](https://feedback.azure.com/forums/903958-azure-container-registry/suggestions/32517127-enable-anonymous-access-to-registries).
+Setting up an Azure container registry for anonymous (public) pull access is currently a preview feature. Please delete [scope map (user) and token resources](https://aka.ms/acr/repo-permissions) from your registry before raising a support ticket (System scope maps can be ignored). To enable public access, please open a support ticket at https://aka.ms/acr/support/create-ticket. For details, see the [Azure Feedback Forum](https://feedback.azure.com/forums/903958-azure-container-registry/suggestions/32517127-enable-anonymous-access-to-registries).
```

Comment:
Considering this is not a common case, I suggest word like below, also put it to the end or make it a separate paragraph. 
```
  If you ever used token based permission control on the registry, please delete [scope map (user) and token resources]...
```

Reply:
@yugangw-msft Yes, I've rephrased the sentence now.